### ISSUE TITLE
Training gets nan after several minutes

### ISSUE THREAD

**Issue by Renaldas111** [d8ahazard/sd_dreambooth_extension#1248]
#######################################################################################################
Initializing Dreambooth
Dreambooth revision: 926ae204ef5de17efca2059c334b6098492a0641
Successfully installed fastapi-0.94.1 gitpython-3.1.31 requests-2.31.0 transformers-4.26.1

Does your project take forever to startup?
Repetitive dependency installation may be the reason.
Automatic1111's base project sets strict requirements on outdated dependencies.
If an extension is using a newer version, the dependency is uninstalled and reinstalled twice every startup.

[+] xformers version 0.0.17 installed.
[+] torch version 2.0.0+cu118 installed.
[+] torchvision version 0.15.1+cu118 installed.
[+] accelerate version 0.18.0 installed.
[+] diffusers version 0.14.0 installed.
[+] transformers version 4.26.1 installed.
[+] bitsandbytes version 0.35.4 installed.

#######################################################################################################
```
I was succesfully training all what I need about 2-3 months ago, with nans appearing, but quite a rarely. I am training in Windows 11 on RXT3060 12G. My training settings are like here:
Initializing bucket counter!
  ***** Running training *****
  Num batches each epoch = 45
  Num Epochs = 100
  Batch Size Per Device = 5
  Gradient Accumulation steps = 1
  Total train batch size (w. parallel, distributed & accumulation) = 5
  Text Encoder Epochs: 100
  Total optimization steps = 22500
  Total training steps = 22500
  Resuming from checkpoint: False
  First resume epoch: 0
  First resume step: 0
  Lora: False, Optimizer: 8bit AdamW, Prec: fp16
  Gradient Checkpointing: True
  EMA: False
  UNET: True
  Freeze CLIP Normalization Layers: False
  LR: 2e-06
  V2: False

I usually train from v1.5-pruned.ckpt model several concepts with no regularization images. Today, after months of non-training I noticed all my trainings end in nan after several minutes. 
Any ideas?

**Comment by swumagic** [comment]
After each update, the requirements.txt must be modified as follows.   
extensions\sd_dreambooth_extension\requirements.txt。it run ok。

accelerate~=0.18.0
bitsandbytes==0.35.4
dadaptation~=3.1
diffusers~=0.16.1
discord-webhook~=1.1.0
fastapi~=0.94.1
gitpython~=3.1.31
lion-pytorch~=0.0.7
Pillow~=9.5.0
tqdm~=4.64.1
transformers~=4.25.1 # > 4.26.x causes issues (db extension #1110)

**Comment by Renaldas111** [comment]
Thanks, I followed your advice!
Now I receive extremely long error message:

**Error running install.py for extension E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension.
Command: "E:\stable-diffusion-webui-W11\venv\Scripts\python.exe" "E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\install.py"
Error code: 1
stdout: If submitting an issue on github, please provide the full startup log for debugging purposes.

Initializing Dreambooth
Dreambooth revision: 926ae204ef5de17efca2059c334b6098492a0641
# Requirement installation exception:
WARNING: Ignoring invalid distribution - (e:\stable-diffusion-webui-w11\venv\lib\site-packages)
WARNING: Ignoring invalid distribution -llow (e:\stable-diffusion-webui-w11\venv\lib\site-packages)
WARNING: Ignoring invalid distribution -illow (e:\stable-diffusion-webui-w11\venv\lib\site-packages)
WARNING: Ignoring invalid distribution -rotobuf (e:\stable-diffusion-webui-w11\venv\lib\site-packages)
WARNING: Ignoring invalid distribution -llow (e:\stable-diffusion-webui-w11\venv\lib\site-packages)
Ignoring tensorboard: markers 'sys_platform == "darwin" and platform_machine == "arm64"' don't match your environment
Ignoring tensorflow-macos: markers 'sys_platform == "darwin" and platform_machine == "arm64"' don't match your environment
Ignoring tensorflow-metal: markers 'sys_platform == "darwin" and platform_machine == "arm64"' don't match your environment
Requirement already satisfied: accelerate~=0.18.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 1)) (0.18.0)
Requirement already satisfied: bitsandbytes==0.35.4 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 2)) (0.35.4)
Collecting dadaptation~=3.1 (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 3))
Using cached dadaptation-3.1-py3-none-any.whl
Collecting diffusers~=0.16.1 (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 4))
Using cached diffusers-0.16.1-py3-none-any.whl (934 kB)
Requirement already satisfied: discord-webhook~=1.1.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 5)) (1.1.0)
Collecting fastapi~=0.94.1 (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 6))
Using cached fastapi-0.94.1-py3-none-any.whl (56 kB)
Collecting gitpython~=3.1.31 (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 7))
Using cached GitPython-3.1.31-py3-none-any.whl (184 kB)
Requirement already satisfied: lion-pytorch~=0.0.7 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 8)) (0.0.7)
Collecting Pillow~=9.5.0 (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 9))
Using cached Pillow-9.5.0-cp310-cp310-win_amd64.whl (2.5 MB)
Requirement already satisfied: tqdm~=4.64.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 10)) (4.64.1)
Requirement already satisfied: transformers~=4.25.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 11)) (4.25.1)
Requirement already satisfied: tensorboard==2.12.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (2.12.0)
Requirement already satisfied: tensorflow==2.12.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from -r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (2.12.0)
Requirement already satisfied: absl-py>=0.4 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (1.4.0)
Requirement already satisfied: grpcio>=1.48.2 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (1.51.3)
Requirement already satisfied: google-auth<3,>=1.6.3 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (2.16.2)
Requirement already satisfied: google-auth-oauthlib<0.5,>=0.4.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (0.4.6)
Requirement already satisfied: markdown>=2.6.8 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (3.4.1)
Requirement already satisfied: numpy>=1.12.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (1.23.5)
Requirement already satisfied: protobuf>=3.19.6 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (3.20.3)
Requirement already satisfied: requests<3,>=2.21.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (2.25.1)
Requirement already satisfied: setuptools>=41.0.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (63.2.0)
Requirement already satisfied: tensorboard-data-server<0.8.0,>=0.7.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (0.7.0)
Requirement already satisfied: tensorboard-plugin-wit>=1.6.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (1.8.1)
Requirement already satisfied: werkzeug>=1.0.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (2.2.3)
Requirement already satisfied: wheel>=0.26 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (0.38.4)
Requirement already satisfied: tensorflow-intel==2.12.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (2.12.0)
Requirement already satisfied: astunparse>=1.6.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (1.6.3)
Requirement already satisfied: flatbuffers>=2.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (23.3.3)
Requirement already satisfied: gast<=0.4.0,>=0.2.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (0.4.0)
Requirement already satisfied: google-pasta>=0.1.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (0.2.0)
Requirement already satisfied: h5py>=2.9.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (3.8.0)
Requirement already satisfied: jax>=0.3.15 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (0.4.8)
Requirement already satisfied: libclang>=13.0.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (16.0.0)
Requirement already satisfied: opt-einsum>=2.3.2 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (3.3.0)
Requirement already satisfied: packaging in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (23.0)
Requirement already satisfied: six>=1.12.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (1.16.0)
Requirement already satisfied: termcolor>=1.1.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (2.2.0)
Requirement already satisfied: typing-extensions>=3.6.6 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (4.5.0)
Requirement already satisfied: wrapt<1.15,>=1.11.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (1.14.1)
Requirement already satisfied: tensorflow-estimator<2.13,>=2.12.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (2.12.0)
Requirement already satisfied: keras<2.13,>=2.12.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (2.12.0)
Requirement already satisfied: tensorflow-io-gcs-filesystem>=0.23.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (0.31.0)
Requirement already satisfied: psutil in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from accelerate~=0.18.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 1)) (5.9.4)
Requirement already satisfied: pyyaml in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from accelerate~=0.18.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 1)) (6.0)
Requirement already satisfied: torch>=1.4.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from accelerate~=0.18.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 1)) (2.0.0+cu118)
Requirement already satisfied: filelock in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from diffusers~=0.16.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 4)) (3.9.0)
Collecting huggingface-hub>=0.13.2 (from diffusers~=0.16.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 4))
Using cached huggingface_hub-0.15.1-py3-none-any.whl (236 kB)
Requirement already satisfied: importlib-metadata in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from diffusers~=0.16.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 4)) (6.3.0)
Requirement already satisfied: regex!=2019.12.17 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from diffusers~=0.16.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 4)) (2022.10.31)
Collecting requests<3,>=2.21.0 (from tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14))
Using cached requests-2.31.0-py3-none-any.whl (62 kB)
Requirement already satisfied: pydantic!=1.7,!=1.7.1,!=1.7.2,!=1.7.3,!=1.8,!=1.8.1,<2.0.0,>=1.6.2 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from fastapi~=0.94.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 6)) (1.10.6)
Requirement already satisfied: starlette<0.27.0,>=0.26.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from fastapi~=0.94.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 6)) (0.26.1)
Requirement already satisfied: gitdb<5,>=4.0.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from gitpython~=3.1.31->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 7)) (4.0.10)
Requirement already satisfied: colorama in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from tqdm~=4.64.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 10)) (0.4.6)
Requirement already satisfied: tokenizers!=0.11.3,<0.14,>=0.11.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from transformers~=4.25.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 11)) (0.13.2)
Requirement already satisfied: smmap<6,>=3.0.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from gitdb<5,>=4.0.1->gitpython~=3.1.31->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 7)) (5.0.0)
Requirement already satisfied: cachetools<6.0,>=2.0.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from google-auth<3,>=1.6.3->tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (5.3.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from google-auth<3,>=1.6.3->tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (0.2.8)
Requirement already satisfied: rsa<5,>=3.1.4 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from google-auth<3,>=1.6.3->tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (4.9)
Requirement already satisfied: requests-oauthlib>=0.7.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from google-auth-oauthlib<0.5,>=0.4.1->tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (1.3.1)
Requirement already satisfied: fsspec in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from huggingface-hub>=0.13.2->diffusers~=0.16.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 4)) (2023.3.0)
Requirement already satisfied: charset-normalizer<4,>=2 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from requests<3,>=2.21.0->tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (3.1.0)
Requirement already satisfied: idna<4,>=2.5 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from requests<3,>=2.21.0->tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (2.10)
Requirement already satisfied: urllib3<3,>=1.21.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from requests<3,>=2.21.0->tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (1.26.14)
Requirement already satisfied: certifi>=2017.4.17 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from requests<3,>=2.21.0->tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (2022.12.7)
Requirement already satisfied: anyio<5,>=3.4.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from starlette<0.27.0,>=0.26.1->fastapi~=0.94.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 6)) (3.6.2)
Requirement already satisfied: sympy in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from torch>=1.4.0->accelerate~=0.18.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 1)) (1.11.1)
Requirement already satisfied: networkx in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from torch>=1.4.0->accelerate~=0.18.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 1)) (3.0)
Requirement already satisfied: jinja2 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from torch>=1.4.0->accelerate~=0.18.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 1)) (3.1.2)
Requirement already satisfied: MarkupSafe>=2.1.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from werkzeug>=1.0.1->tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (2.1.2)
Requirement already satisfied: zipp>=0.5 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from importlib-metadata->diffusers~=0.16.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 4)) (3.15.0)
Requirement already satisfied: sniffio>=1.1 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from anyio<5,>=3.4.0->starlette<0.27.0,>=0.26.1->fastapi~=0.94.1->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 6)) (1.3.0)
Requirement already satisfied: ml-dtypes>=0.0.3 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from jax>=0.3.15->tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (0.1.0)
Requirement already satisfied: scipy>=1.7 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from jax>=0.3.15->tensorflow-intel==2.12.0->tensorflow==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 15)) (1.10.1)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from pyasn1-modules>=0.2.1->google-auth<3,>=1.6.3->tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (0.4.8)
Requirement already satisfied: oauthlib>=3.0.0 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib<0.5,>=0.4.1->tensorboard==2.12.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 14)) (3.2.2)
Requirement already satisfied: mpmath>=0.19 in e:\stable-diffusion-webui-w11\venv\lib\site-packages (from sympy->torch>=1.4.0->accelerate~=0.18.0->-r E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\requirements.txt (line 1)) (1.3.0)
WARNING: Ignoring invalid distribution - (e:\stable-diffusion-webui-w11\venv\lib\site-packages)
WARNING: Ignoring invalid distribution -llow (e:\stable-diffusion-webui-w11\venv\lib\site-packages)
WARNING: Ignoring invalid distribution -illow (e:\stable-diffusion-webui-w11\venv\lib\site-packages)
WARNING: Ignoring invalid distribution -rotobuf (e:\stable-diffusion-webui-w11\venv\lib\site-packages)
WARNING: Ignoring invalid distribution -llow (e:\stable-diffusion-webui-w11\venv\lib\site-packages)
Installing collected packages: requests, Pillow, dadaptation, huggingface-hub, gitpython, fastapi, diffusers
Attempting uninstall: requests
Found existing installation: requests 2.25.1
Uninstalling requests-2.25.1:
Successfully uninstalled requests-2.25.1
Attempting uninstall: Pillow
Found existing installation: Pillow 9.4.0
Uninstalling Pillow-9.4.0:
Successfully uninstalled Pillow-9.4.0
ERROR: Could not install packages due to an OSError: [WinError 5] Access is denied: 'E:\\stable-diffusion-webui-W11\\venv\\Lib\\site-packages\\~%l\\_imaging.cp310-win_amd64.pyd'
Check the permissions.
[notice] A new release of pip is available: 23.1 -> 23.1.2
[notice] To update, run: E:\stable-diffusion-webui-W11\venv\Scripts\python.exe -m pip install --upgrade pip

stderr: E:\stable-diffusion-webui-W11\venv\lib\site-packages\pkg_resources\__init__.py:123: PkgResourcesDeprecationWarning: llow is an invalid version and will not be supported in a future release
  warnings.warn(
A matching Triton is not available, some optimizations will not be enabled.
Error caught was: No module named 'triton'
E:\stable-diffusion-webui-W11\venv\lib\site-packages\pytorch_lightning\utilities\distributed.py:258: LightningDeprecationWarning: `pytorch_lightning.utilities.distributed.rank_zero_only` has been deprecated in v1.8.1 and will be removed in v2.0.0. You can import it from `pytorch_lightning.utilities` instead.
  rank_zero_deprecation(
loading stable diffusion model: ImportError
Traceback (most recent call last):
  File "E:\stable-diffusion-webui-W11\modules\sd_models.py", line 419, in get_sd_model
    load_model()
  File "E:\stable-diffusion-webui-W11\modules\sd_models.py", line 436, in load_model
    from modules import lowvram, sd_hijack
  File "E:\stable-diffusion-webui-W11\modules\sd_hijack.py", line 5, in <module>
    import modules.textual_inversion.textual_inversion
  File "E:\stable-diffusion-webui-W11\modules\textual_inversion\textual_inversion.py", line 18, in <module>
    from modules import shared, devices, sd_hijack, processing, sd_models, images, sd_samplers, sd_hijack_checkpoint
  File "E:\stable-diffusion-webui-W11\modules\processing.py", line 18, in <module>
    from modules.sd_hijack import model_hijack
ImportError: cannot import name 'model_hijack' from partially initialized module 'modules.sd_hijack' (most likely due to a circular import) (E:\stable-diffusion-webui-W11\modules\sd_hijack.py)


Stable diffusion model failed to load
+--------------------- Traceback (most recent call last) ---------------------+
| E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\install.py |
| :35 in <module>                                                             |
|                                                                             |
|   32                                                                        |
|   33 from postinstall import actual_install                                 |
|   34                                                                        |
| > 35 actual_install()                                                       |
|   36                                                                        |
|                                                                             |
| E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\postinstal |
| l.py:41 in actual_install                                                   |
|                                                                             |
|    38     print("Initializing Dreambooth")                                  |
|    39     print(f"Dreambooth revision: {revision}")                         |
|    40                                                                       |
| >  41     install_requirements()                                            |
|    42                                                                       |
|    43     check_xformers()                                                  |
|    44                                                                       |
|                                                                             |
| E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\postinstal |
| l.py:87 in install_requirements                                             |
|                                                                             |
|    84     except subprocess.CalledProcessError as grepexc:                  |
|    85         error_msg = grepexc.stdout.decode()                           |
|    86         print_requirement_installation_error(error_msg)               |
| >  87         raise grepexc                                                 |
|    88                                                                       |
|    89                                                                       |
|    90 def check_xformers():                                                 |
|                                                                             |
| E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\postinstal |
| l.py:75 in install_requirements                                             |
|                                                                             |
|    72     transformers_version = importlib_metadata.version("transformers") |
|    73                                                                       |
|    74     try:                                                              |
| >  75         pip_install("-r", req_file)                                   |
|    76                                                                       |
|    77         if has_diffusers and has_tqdm and Version(transformers_versio |
|    78             print()                                                   |
|                                                                             |
| E:\stable-diffusion-webui-W11\extensions\sd_dreambooth_extension\postinstal |
| l.py:53 in pip_install                                                      |
|                                                                             |
|    50                                                                       |
|    51                                                                       |
|    52 def pip_install(*args):                                               |
| >  53     output = subprocess.check_output(                                 |
|    54         [sys.executable, "-m", "pip", "install"] + list(args),        |
|    55         stderr=subprocess.STDOUT,                                     |
|    56         )                                                             |
|                                                                             |
| C:\Program Files\Python310\lib\subprocess.py:420 in check_output            |
|                                                                             |
|    417             empty = b''                                              |
|    418         kwargs['input'] = empty                                      |
|    419                                                                      |
| >  420     return run(*popenargs, stdout=PIPE, timeout=timeout, check=True, |
|    421                **kwargs).stdout                                      |
|    422                                                                      |
|    423                                                                      |
|                                                                             |
| C:\Program Files\Python310\lib\subprocess.py:524 in run                     |
|                                                                             |
|    521             raise                                                    |
|    522         retcode = process.poll()                                     |
|    523         if check and retcode:                                        |
| >  524             raise CalledProcessError(retcode, process.args,          |
|    525                                      output=stdout, stderr=stderr)   |
|    526     return CompletedProcess(process.args, retcode, stdout, stderr)   |
|    527                                                                      |
+-----------------------------------------------------------------------------+
CalledProcessError: Command
'['E:\\stable-diffusion-webui-W11\\venv\\Scripts\\python.exe', '-m', 'pip',
'install', '-r',
'E:\\stable-diffusion-webui-W11\\extensions\\sd_dreambooth_extension\\requireme
nts.txt']' returned non-zero exit status 1.**

Despite this Dreambooth loads and trains without going nan after several minutes! :)
Looks it is better not to touch anything and leave this error :)
